config/config.go

```diff
@@ -208,6 +210,10 @@ var DefaultRateLimiterConfig = &localhttp.RateLimiterConfig{
 	},
 }
 
+var DefaultBandwidthLimiterConfig = &localhttp.BandWidthLimiterConfig{
+	Enable: false,
```

Comment:
Maybe you need to update the default configuration synchronously at config/config_template.toml.

Reply:
added